src/contrib/mlir/compiler/compiler.cpp

```diff
@@ -729,11 +736,10 @@ void MLIRCompiler::bindArguments(std::vector<void*>& externalTensors)
     m_externalTensors = &externalTensors;
 
     // Create list with a type-erased double pointer for each invocation arguments.
-    // We currently use 'allocateMemrefArgs', which creates a
-    // SmallVector<StaticFloatMemref*>. StaticFloatMemref is just a struct with the
-    // actual pointer to the data.
+    // We currently use 'allocateMemrefArgs', which creates the arguments list per call ABI (see
+    // comment below).
+    // StaticFloatMemref is just a struct with the actual pointer to the data.
```

Comment:
Not sure I can understand the `StaticFloatMemref` comment now. Could we elaborate this a bit more, add more info about the ABI here or rephrase this two comments in some way?

Reply:
790 has an example of the expected args layout in memory.